contracts/peripherals/Timelock.sol

```diff
@@ -152,27 +157,63 @@ contract Timelock is ITimelock {
         uint256 _minProfitTime,
         bool _hasDynamicFees
     ) external onlyAdmin {
-        require(_taxBasisPoints < MAX_FEE_BASIS_POINTS, "Timelock: invalid _taxBasisPoints");
-        require(_stableTaxBasisPoints < MAX_FEE_BASIS_POINTS, "Timelock: invalid _stableTaxBasisPoints");
-        require(_mintBurnFeeBasisPoints < MAX_FEE_BASIS_POINTS, "Timelock: invalid _mintBurnFeeBasisPoints");
-        require(_swapFeeBasisPoints < MAX_FEE_BASIS_POINTS, "Timelock: invalid _swapFeeBasisPoints");
-        require(_stableSwapFeeBasisPoints < MAX_FEE_BASIS_POINTS, "Timelock: invalid _stableSwapFeeBasisPoints");
-        require(_marginFeeBasisPoints < MAX_FEE_BASIS_POINTS, "Timelock: invalid _marginFeeBasisPoints");
-        require(_liquidationFeeUsd < 10 * PRICE_PRECISION, "Timelock: invalid _liquidationFeeUsd");
+        marginFeeBasisPoints = _marginFeeBasisPoints;
 
         IVault(_vault).setFees(
             _taxBasisPoints,
             _stableTaxBasisPoints,
             _mintBurnFeeBasisPoints,
             _swapFeeBasisPoints,
             _stableSwapFeeBasisPoints,
-            _marginFeeBasisPoints,
+            MAX_FEE_BASIS_POINTS,
             _liquidationFeeUsd,
             _minProfitTime,
             _hasDynamicFees
         );
     }
 
+    function enableLeverage(address _vault) external override onlyAdminOrHandler {
+        IVault vault = IVault(_vault);
+
+        if (hasVaultUtils) {
+            vault.setIsLeverageEnabled(true);
```

Comment:
On Avalanche, hasVaultUtils will be true, so we can just set isLeverageEnabled to false and it will disable both increasePosition and decreasePosition

On Arbitrum, hasVaultUtils will be false, since we can't use vaultUtils to disable leverage, so we set the margin fee to the max fee instead